### PR TITLE
Updated travis file to also run windows builds

### PR DESCRIPTION
**Pull request by rofinn**

### CODE REVIEW

**Comment by rofinn** [comment]
NOTE: The failure is for cancelled jobs that were sent to the old travis-ci.org location (I think it was a bug in the github API?). Eric has disabled the legacy travis as it appears to be a bad migration.

File: .travis.yml [hunk 1]
```diff
@@ -3,27 +3,38 @@ language: julia
...
+#script:
+#  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+#  - julia -e 'Pkg.clone(pwd()); Pkg.build("Impute"); Pkg.test("Impute"; coverage=true)'
 after_success:
   - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
+jobs:
+  fast_finish: true
+  allow_failures:
+    - julia: nightly
+    - arch: x86   # Issues installing dependencies on x86
```
Comment (oxinabox) [inline .travis.yml:26]: Can we add an URL for the issue number here?

File: .travis.yml [hunk 2]
```diff
@@ -3,27 +3,38 @@ language: julia
...
   - osx
+  - windows
 julia:
-  - 1.0
-  - 1.1
-  - 1.2
-  - 1.3
-  - 1.4
+  - 1.0   # LTS
+  - 1.3   # Release Invenia uses
```
Comment (oxinabox) [inline .travis.yml:9]: Do we want to add:
```suggestion
  - 1  # stable
```

**Review by oxinabox — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (rofinn) [reply .travis.yml:9]: Yes, but probably in a separate PR because I don't think the specimen files generated for 1.3 work properly on 1.5.

Comment (rofinn) [reply .travis.yml:9]: That's part of what #80 is about

**Comment by rofinn** [comment]
Looks like we're moving away from travis too now. I'm gonna close this and open another using github actions. Sorry for the noise and thanks for reviewing @oxinabox

**Pull request closed without merging**